README.md

```diff
@@ -145,37 +124,45 @@ for _, client := range clients {
 ### Subscribing to all presence messages
 
 ```go
-sub, err := channel.Presence.Subscribe()
+unsubscribe, err = channel.Presence.SubscribeAll(ctx, func(msg *ably.PresenceMessage) {
+	fmt.Println("Presence event: action=%v data=%v", msg.Action, msg.Data)
```

Comment:
This should use `fmt.Printf`

Reply:
Done at f2b682779470cb2b98833d3f702026984efdfe27.